packages/zipkin-instrumentation-express/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import {Tracer} from "zipkin"
+import {Handler} from "express"
+
+/**
+ * When a request comes in, creates ServerRecv annotation and then passes
```

Comment:
what's the usual strategy for docs? put them in typescript, then make a note in the source javascript to look at the ts?

Reply:
I don’t know what people normally do. If you want I can move it to the js file

Reply:
this is fine for now.. thx